### ISSUE TITLE
JSX element inside of attribute causes highlighting to break

### ISSUE THREAD

**Issue by cesarandreu** [babel/babel-sublime#163]
Looking at react-bootstrap's [OverlayTriggerSpec](https://raw.githubusercontent.com/react-bootstrap/react-bootstrap/583febb82a7286f23d099288c01c46061e61381e/src/OverlayTrigger.js), I found that if you have a JSX element inside of a JSX element's attribute, it'll break syntax highlighting.

Reduced example:

```js
import React from 'react';
import ReactTestUtils from 'react/lib/ReactTestUtils';
import ReactDOM from 'react-dom';

import OverlayTrigger from '../src/OverlayTrigger';
import Popover from '../src/Popover';
import Tooltip from '../src/Tooltip';

describe('OverlayTrigger', function() {
  it('Should create OverlayTrigger element', function() {
    const instance = ReactTestUtils.renderIntoDocument(
      <OverlayTrigger overlay={<div>test</div>}>
        <button>button</button>
      </OverlayTrigger>
    );
    const overlayTrigger = ReactDOM.findDOMNode(instance);
    assert.equal(overlayTrigger.nodeName, 'BUTTON');
  });
});
```

Screenshot:

![http://i.imgur.com/bBZZldE.png](http://i.imgur.com/bBZZldE.png)

Version information:

* Sublime Text, Dev Channel, build 3095
* Babel v8.0.1
* Package Control v3.0.1

**Comment by zertosh** [comment]
Awesome report! Thanks! Just like https://github.com/babel/babel-sublime/issues/162. I just fixed it https://github.com/babel/babel-sublime/commit/e7b41c90b09288a8838d0de8a52b0a4c09b5c46d and published it as v8.0.2 – sorry about that.

**Comment by cesarandreu** [comment]
Oh wow, that was super fast, thanks for the speedy fix!
